libs/deer/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "deer",
+  "version": "0.0.0-private",
+  "private": true
+}
```

Comment:
Could we [`exclude`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields) this file in `Cargo.toml` please?

Reply:
Good point! Addressed in https://github.com/hashintel/hash/pull/1858/commits/e33821031577b682a1f5d36b6b69320360c02445

Reply:
Thanks!